Add userTracking toggle

simofacc opened the pull request:
Add enableUserTracking boolean to app.js to easily turn tracking on and
off.

Related to #226

Commit:
Add userTracking toggle

Add enableUserTracking boolean to app.js to easily turn tracking on and
off.

leameow committed:
Update README.md with css dir for compass command

On some system it seems like default css directory is `stylesheet` instead of `css`.

leameow committed:
Added settings button

leameow committed:
Fixed hidden popular page

leameow committed:
Changed id of all page view to movies-list

leameow committed:
Added missing settings icon

leameow committed:
Added Settings page

leameow committed:
Added a settings page with a setting to set the download folder

leameow committed:
Localize new string in en and fr

leameow committed:
Aligne settings icon with fullscreen icon

leameow committed:
Add option to not clear cache after movie

leameow commented:
I started adding a settings page (https://github.com/popcorn-time/popcorn-app/pull/660). Perhaps this could be added there as well.

simofacc commented:
If we want to let the user choose if he wants to be tracked or not then it's fine, but otherwise it might not be needed.

In any case you should use a checkbox with the label "Help us improve the application" or something like that since the tracking is mainly used to track user actions.

Commit:
Merge remote-tracking branch 'upstream/master'

leameow commented:
I believe it is a good practise to let the user knows his action are tracked by default and giving people the option to opt-out is quite important in my opinion. Especially in the case of Popcorntime where not everyone might like the fact they are sending usage data about illegal copyright infringment...

simofacc commented:
Yes I agree completely.

Commit:
Add Maltese language

Translate interface string to the Maltese language
Add maltese flag to the images folder and in the css code

leameow committed:
Torrent source is now configurable

leameow committed:
Merge branch 'feature-settings'

leameow committed:
Add Yts scraper

leameow committed:
Remove broken updater

leameow committed:
Disable tracking

leameow committed:
Remove updater.js reference

leameow committed:
Merge branch 'bugfix'

leameow committed:
Merge branch 'master' into feature-trackers

Conflicts:
	index.html
	js/frontend/controllers/search.js

leameow committed:
Add yts scraper

leameow committed:
Updated movie parsing with last beta

danielfariati committed:
removing old popcorntime repository references

leameow committed:
Merge pull request #2 from danielfariati/master

removing old popcorntime repository references

leameow committed:
Merge branch 'master' of github.com:isra17/popcorn-app

leameow committed:
Merge branch 'feature-trackers'

Commit:
Fix search and add support for larger images

leameow committed:
Add moviedb to get metadata

leameow committed:
Add dependancy patch

leameow committed:
Merge pull request #8 from maxmcd/master

Fix search and add support for larger images

Commit:
Use new fork
